test/new-e2e/tests/agent-platform/common/svc-manager/systemctl.go

```diff
@@ -25,6 +25,7 @@ func NewSystemctlSvcManager(vmClient client.VM) *SystemCtlSvcManager {
 // Status returns status from systemctl
 func (s *SystemCtlSvcManager) Status(service string) (string, error) {
 	return s.vmClient.ExecuteWithError(fmt.Sprintf("systemctl status --no-pager %s.service", service))
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
```